mvp-spec/operator-api.md

```diff
@@ -451,18 +392,66 @@ cat response-operatorO.json body-id.json | npx json --merge
   "timestamp": 1639059692793,
   "signature": "message_signature_xyz1234",
   "body": {
-    "version": 1,
-    "type": "prebid_id",
-    "value": "7435313e-caee-4889-8ad7-0acd0114ae3c",
-    "source": {
-      "domain": "operator0.com",
-      "timestamp": 1639643112,
-      "signature": "12345_signature"
+    "newIdentifier": {
+      "version": 1,
+      "type": "prebid_id",
+      "value": "560cead0-eed5-4d3f-a308-b818b4827979",
+      "source": {
+        "domain": "operator0.com",
+        "timestamp": 1639643110,
+        "signature": "prebid_id_signature_xyz12345"
+      }
     }
   }
 }
 ```
 
+##### Response signature
+
+Signature of the concatenation of:
+
+```
+sender + '\u2063' +
+receiver + '\u2063' +
+newIdentifier.source.signature + '\u2063' +
+timestamp
+```
+
+### GET /v1/json/verify3PC
+
+This endpoint is very "technical" and used only when a first attempt to read Prebid cookies via REST failed.
```

Comment:
I guess that at this level everything is technical. :D 
Put simply: "This endpoint is used when a first attempt to read..."